logic/atrium-logic/src/commonMain/kotlin/ch/tutteli/atrium/logic/impl/DefaultAnyAssertions.kt

```diff
@@ -98,4 +98,11 @@ class DefaultAnyAssertions : AnyAssertions {
                 .build()
         ).build()
     }
+
+
+    override fun <T, TSub : Any> notToBeAnInstanceOf(container: AssertionContainer<T>, expected: KClass<TSub>): SubjectChangerBuilder.ExecutionStep<T, Boolean> =
+        container.changeSubject.reportBuilder()
+            .checkCast(expected)
+            .build()
```

Comment:
Since we don't need a transformation, it suffices to use an implementation similar to `toBe` above